.idea/modules.xml

```diff
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/CardiorespiratoryFitness-Android.iml" filepath="$PROJECT_DIR$/CardiorespiratoryFitness-Android.iml" />
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```

Comment:
This was auto added by android studio not sure why it added a duplicate line.  I'm wondering whether this file should be in .gitignore?

Reply:
It doesn't seem to hurt to add modules to .gitignore (other than maybe AndroidStudio first load) but this is supposedly one of the /.idea/ files that can be checked into source control